docs/src/pages/guides/deploy.md

```diff
@@ -182,26 +182,30 @@ jobs:
 
 ## Netlify
 
-In your codebase, make sure you have a [`.nvmrc`](https://github.com/nvm-sh/nvm#nvmrc) file with `node v14.15.1` in it.
-
 You can configure your deploy in two ways, via the Netlify website or with the `netlify.toml` file.
 
-With the `netlify.toml` file, add it at the top level of your project with the following settings:
+## `netlify.toml` file
+
+
+Create a new `netlify.toml` file it at the top level of your project repository with the following settings:
 
 ```toml
 [build]
   command = "npm run build"
   publish = "dist"
 ```
 
-Then, set up a new project on [Netlify](https://netlify.com) from your chosen Git provider.
+Push the new `netlify.toml` file up to to your hosted git repository. Then, set up a new project on [Netlify](https://netlify.com) using you project repo.
 
-If you don't want to use the `netlify.toml`, when you go to [Netlify](https://netlify.com) and set up a new project from Git, input the following settings:
+## Netlify Website UI
+
+You can skip the `netlify.toml` file and go directly to [Netlify](https://netlify.com) to configure your project. Netlify should now detect Astro projects automatically and pre-fill the configuration for you. Make sure that the the following settings are entered before hitting the "Deploy" button:
 
 - **Build Command:** `astro build` or `npm run build`
 - **Publish directory:** `dist`
 
-Then hit the deploy button.
+**Note:** If using an older build image, make sure that you have a [`.nvmrc`](https://github.com/nvm-sh/nvm#nvmrc) file with `node v14.15.1` in it. This step is no longer required by default.
```

Comment:
Is it worth mentioning that instead of the .nvmrc file, you can instead set an environment variable NODE_VERSION: 14.15.1 ? That's how I ended up getting mine set up months ago (because. reasons.) and it works just fine, but I don't know if that's not the "preferred" way to do it.

Reply:
Also, and maybe this is a question for @FredKSchott, now that Netlify is defaulting to a higher version than Astro needs (vs a lower version than Astro needs) and not every project needs to be "upgraded" to 14.15.1 by default, do we know which specific versions of Astro will work with the default Node 16, and which would need a "downgrade" to 14.15.1? If so, can we mention the actual version number in the docs? (Related note to self, I guess I should remove my environment variable on Astro 0.20.x now?)

Reply:
Good call out! Here are the versions that we support

```
  "engines": {
    "node": "^12.20.0 || ^14.13.1 || >=16.0.0",
    "npm": ">=6.14.0"
  }
```

Astro is pretty good about warning you if you run on an incompatible version, and will tell you which versions to upgrade to if it can't run. I think we can recommend a version here, and then if someone wants to experiment with other versions they can feel safe doing that knowing that Astro will give them some guidance if they mess up.

+1 for the environment variable though!